docs/contributing.rst

```diff
@@ -24,8 +24,7 @@ Getting started
 
 .. code-block::
 
-    pip install -r requirements_dev.txt
-    pip install -e .
+    pip install -e . -r requirements_dev.txt
```

Comment:
why? I know that we can unite them, but actually they do different things and I want highlight that, otherwise user can miss `.` sign